recipe/wordpress.php

```diff
@@ -9,15 +9,27 @@
 
 require_once __DIR__ . '/common.php';
 
+set('shared_files', ['wp-config.php']);
+set('shared_dirs', ['wp-content/uploads']);
+
 /**
- * Main task
+ * Chown files to correct web server user
+ * for your OS, so uploads work.
  */
+task('deploy:chown', function () {
+    run('chown -R www-data:www-data ' . env('deploy_path'));
```

Comment:
You are right, it's a better solution to use `writable` task.

WordPress has a quirk that disables the plugin/theme editor and automatic updates if WP core files are not owned by the web server process. However, in this case it shouldn't matter because if you are using Deployer you probably don't want people to go and edit files or update through the web interface anyway.